devel/addon/haproxy/install.sh

```diff
@@ -30,15 +30,31 @@ helm repo add haproxy-ingress https://haproxy-ingress.github.io/charts
 helm repo update
 
 export NAMESPACE="haproxy-ingress"
-export VERSION="0.13.0-beta.2"
+export CONFIG_MAP="haproxy-ingress-config"
+export VERSION="0.13.4"
+
+cat <<EOYAML | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${CONFIG_MAP}
+  namespace: ${NAMESPACE}
+data:
+  max-connections: "250"
```

Comment:
> this works around a limit on file descriptors which we encounter in kind
in CI. newer kind images impose a limit of 1024 file descriptors which
isn't trivial to change; haproxy seems to try to request just over 2*n
file descriptors where n is the max number of connections; as such, if
we limit max-connections to 250 we should be comfortably within the
limit

Awesome debugging work 👍🏼